engines/onnxruntime/onnxruntime-engine/src/main/java/ai/djl/onnxruntime/engine/OrtSymbolBlock.java

```diff
@@ -85,10 +85,18 @@ protected NDList forwardInternal(
         Map<String, OnnxTensor> container = new ConcurrentHashMap<>();
         // forward
         try (OrtNDManager sub = (OrtNDManager) manager.newSubManager()) {
-            // feed data in to match names
-            for (int i = 0; i < inputNames.size(); ++i) {
-                OrtNDArray ortNDArray = sub.from(inputs.get(i));
-                container.put(inputNames.get(i), ortNDArray.getTensor());
+            // If input data has name
+            if (inputs.get(0).getName() != null) {
```

Comment:
We should not assume all input has name

Reply:
User should either provide all names to the NDArray or provides no name